mod/discussions/views/default/resources/discussion/all.php

```diff
@@ -3,8 +3,6 @@
 elgg_pop_breadcrumb();
 elgg_push_breadcrumb(elgg_echo('discussion'));
 
-elgg_register_title_button();
```

Comment:
This change allows accessing all discussions page without providing UI for user-centered discussions by default.